consensus/types/src/chain_spec.rs

```diff
@@ -2020,10 +2008,7 @@ fn default_max_blobs_by_root_request() -> usize {
 }
 
 fn default_data_columns_by_root_request() -> usize {
```

Comment:
Some of these functions feel like they'd benefit from using `LazyLock<usize>` but if they're only used a couple times then whatever.